tools/tcpretrans_example.txt

```diff
@@ -44,6 +44,20 @@ TIME     PID    IP LADDR:LPORT          T> RADDR:RPORT          STATE
 See the "L>" in the "T>" column. These are attempts: the kernel probably
 sent a TLP, but in some cases it might not have been ultimately sent.
 
+To spot heavily retransmitting flows quickly one can use the -c flag. It will
+count occurring retransmits per flow. 
+
+# python tcpretrans.py -c
+Tracing retransmits ... Hit Ctrl-C
+^C
+LADDR:LPORT                 RADDR:RPORT          RETRANSMITS
+192.168.10.50:60366  <-> 172.217.21.194:443         700
+192.168.10.50:666    <-> 172.213.11.195:443         345    
+192.168.10.50:366    <-> 172.212.22.194:443         211
```

Comment:
So, something's odd with the column alignment here -- I'd expect the numbers to either start with the first letter of the column header, or end with the last letter of the column heading, but right now it looks a bit random.